.github/workflows/ubuntu.yml

```diff
@@ -187,6 +184,15 @@ jobs:
         with:
```

Comment:
I'd also exclude the combination Ubuntu 20.04 with Python 3.6. I think it makes sense to test both Ubuntu versions only with their default Python version.

Reply:
I don't understand how this goes with our goal of supporting Python 3.9. We decided to test all Python versions within their lifespan. Currently, we do this by testing the oldest and latest version supported in Fast Downward, which is 3.6 and 3.8. With the compilers, we test the default compiler on the older Ubuntu and the default and the newest version on the newer Ubuntu release. This just mimics this behavior.

Reply:
If this goes against our rules, we need to rethink them. I believe testing the same Python version on different Ubuntu versions is wasting CPU cycles. I also wouldn't compile the planner twice for the same GCC version on different Ubuntu versions.